fullstory-segment-middleware/src/test/java/com/fullstorydev/fullstory_segment_middleware/FSPropertiesUtils.java

```diff
@@ -0,0 +1,388 @@
+package com.fullstorydev.fullstory_segment_middleware;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FSPropertiesUtils {
```

Comment:
I would be a good idea to add a comment to this class that describes how it returns expected transformations when adding type suffixes to the properties of ecommerce objects outlines in the Segment ecommerce spec.